### PR TITLE
Added offset example

### PR DESCRIPTION
**Pull request by gvarao**
Added offset example and message for green terms

### CODE REVIEW

**Comment by prashant3167** [comment]
jenkins please retry a build

File: juniper_official/Solutions/Icmp-outlier/icmp.rule [hunk 1]
```diff
@@ -103,15 +103,24 @@
...
+                 * e.g. if sensor frequency is 1m and offset is set to 2 then 
+                 * trigger frequency is 2 minutes.
+                 */
+                frequency 1offset;
                 /*
                  * Sets color to red and sends out an anomaly notification when
-                 * the packet ($packet-loss-percent) is 100.
+                 * the packet ($packet-loss-percent) is 100 continuously for
+                 * the offset duration. The offset value is a multiple of the
+                 * configured sensor frequency or a configured sensor profile
```
Comment (ConstrainedRandomVar) [inline juniper_official/Solutions/Icmp-outlier/icmp.rule:118]: Can you please include a full stop here?

Comment (gvarao) [reply juniper_official/Solutions/Icmp-outlier/icmp.rule]: Added pullstop

Comment (gvarao) [reply juniper_official/Solutions/Icmp-outlier/icmp.rule]: Please check now

**Pull request merged**